.bumpversion.cfg

```diff
@@ -0,0 +1,6 @@
+[bumpversion]
```

Comment:
I think we should look for a similar configuration that the one we have already in place e.g.
https://github.com/ome/omero-py/blob/master/.bumpversion.cfg

Reply:
I have revised the bump2version configuration file to match the one we use.